client/common_models.go

```diff
@@ -0,0 +1,13 @@
+package client
+
+type User struct {
```

Comment:
This is the only struct (I could find) that is common across many files.
Decided to create a common_models.go file for this use-case. (It shouldn't bloat as model.go did).